docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
       - POSTGRES_PASSWORD=postgres
 
   plausible_events_db:
-    image: yandex/clickhouse-server:21.3.2.5
+    image: clickhouse/clickhouse-server:22.4.2.1
```

Comment:
It actually works fine with `22.6.1` for me, but I admit there potentially can be breaking changes that aren't on the surface.

Reply:
I'm also using [`22.6-alpine`](https://github.com/ruslandoga/hosting/pull/1) on my instance. @paskal can it be bumped up to `22.6`? Are there any reasons not to use alpine version?

Reply:
Indeed, I'll bump it. I stayed conservative in changing versions as back in April, and I thought it would help me to merge my request faster, but it turns out I was wrong.